File: Python/Minor_Assignment_3/Q9.py
def binaryToDecimal(n):
    decimal=0
    i=0
    while n!=0:
        b=n%10
        decimal+=b*pow(2,i)
        n=n//10
        i+=1
    return decimal

def decimalToBinary(n):
    binary=0
    i=1
    while n!=0:
        b=n%2
        binary+=b*i
        n=n//2
        i*=10
    return binary

num = int(input("Enter a number: "))
btd=decimalToBinary(num)
dtb=binaryToDecimal(btd)
print("Binary of",num,"is",btd)
print("Decimal of",btd,"is",dtb)

'''
Output:
Enter a number: 12
Binary of 12 is 1100
Decimal of 1100 is 12
'''

File: PPWC/Minor_Assignment_7/Q9.c
#include <stdio.h>
struct s{int *p;};
int main(){int a=200;struct s s1;
s1.p=&a; *(s1.p)=*(s1.p)+100;
printf("%d %d\n",a,*(s1.p));
return 0;}

/*
Output--
300 300
*/